.github/ISSUE_TEMPLATE/release-plan.md

```diff
@@ -30,10 +32,21 @@ assignees: 'Guite, craigh'
 
 - [ ] Start release process in core manager at <https://ziku.la>
   - [ ] Create QA ticket
-  - [ ] Create release
-  - [ ] Copy assets (note: it now downloads assets from the chosen GitHub Actions build, since Jenkins has been removed)
+  - [ ] Create tag for core project
+  - [ ] Create tag for all slave repositories
+- [ ] Wait a bit for packagist to update
+- [ ] Update version number in distribution's `composer.json` and update `composer.lock`
+  - [ ] Build distribution (final artifacts) **TODO: add GitHub action workflow for this**
+- [ ] Continue with release process in core manager
+  - [ ] Create tag for distribution project
+  - [ ] Let core manager download the distribution artifacts
+    - it currently downloads assets from the chosen GitHub Actions build from core project **to be changed**
+  - [ ] Create core release (using the previously created tag)
+    - [ ] Push the distribution build artifacts as assets to the core release
+  - [ ] Create distribution release (using the previously created tag)
+    - [ ] Push the distribution build artifacts as assets to the core release
   - [ ] Update core version (this step is currently unused)
```

Comment:
does this mean incrementing the core version post-release? like to 3.0.1 in current case?

Reply:
yes; I am unsure if this makes sense at all because there are typically more things to do, like amending changelogs, creating a corresponding milestone, sorting/assigning issues, etc.

Reply:
still - it is a good first step if it is easily done.

Reply:
also - below in old line 55 it says "Increment version in Kernel class" so perhaps this line is not about incrementing post-release?

Reply:
line 55 is a manual step which is currently needed because the last part of:

> - [ ] Update core version (this step is currently unused)